### PR TITLE
Test Cypress in Chrome 80, FF 72, Debian 10

### PR DESCRIPTION
**Pull request by jennifer-shehane**
- Closes 

### Additional details

The intent of this PR was to only test Chrome 80 upgrade, but the available docker image is `cypress/browsers:node12.8.1-chrome80-ff72`, so this actually is updating Firefox too. I'm open to somehow only bumping Chrome 80, but not sure the best way. 

Ok, well, this automatically fails with yarn:

```
yarn run v1.21.1
$ node scripts/check-node-version.js
🛑 .node-version specified 12.8.1

but current Node is 13.6.0
```

### CODE REVIEW

**Comment by flotwig** [comment]
Trippy, why are the unit-tests still failing with errors caused by a Node version upgrade...? It's back to 12.8.1 in this PR, but these errors are from a 13.x upgrade: https://circleci.com/gh/cypress-io/cypress/258533?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by flotwig** [comment]
After reverting and verting the Docker images, I can confirm that changing the image seems to cause these Node failures, even though the Node version has not changed... super weird, oh well. I'll just fix them.

**Comment by flotwig** [comment]
SSL errors are from this: https://stackoverflow.com/a/58521973/3474615

Apparently this happened in Node 10.x, so idk why we are only just now seeing failures.

File: packages/https-proxy/lib/ca.coffee [hunk 1]
```diff
@@ -176,7 +176,7 @@ class CA
     hosts = [].concat(hosts)
 
     mainHost   = hosts[0]
-    keysServer = pki.rsa.generateKeyPair(1024)
+    keysServer = pki.rsa.generateKeyPair(2048)
```
Comment (flotwig) [inline packages/https-proxy/lib/ca.coffee]: see if this can be 1024

File: packages/https-proxy/lib/ca.coffee [hunk 2]
```diff
@@ -135,7 +135,7 @@ class CA
     sn
 
   generateCA: ->
```
Comment (flotwig) [inline packages/https-proxy/lib/ca.coffee]: need a way to ensure that the CA is recreated, older usersof cypress will have it cached

File: packages/https-proxy/lib/ca.coffee (hunk 1 above)
Comment (flotwig) [inline packages/https-proxy/lib/ca.coffee]: get a rough idea of time change here

File: packages/https-proxy/lib/ca.coffee (hunk 2 above)
Comment (flotwig) [inline packages/https-proxy/lib/ca.coffee]: we can do this on cypress start since it could be a lil slow

Comment (flotwig) [reply packages/https-proxy/lib/ca.coffee]: well, it's already done on server start here:

https://github.com/cypress-io/cypress/blob/e4cd4df1e8389ad5ae2c5aea13a079358055fa68/packages/server/lib/server.coffee#L266-L272

File: packages/https-proxy/lib/ca.coffee (hunk 1 above)
Comment (flotwig) [reply packages/https-proxy/lib/ca.coffee]: added some `console.time` statements to `ca_spec`, here are the measurements for different key sizes:

cert + ca bits | time spent creating CA | time spent generating 1 domain keypair
---- | ---- | ----
512 | 13.967ms | 14.760ms
1024 | 21.039ms | 38.133ms
2048 | 82.628ms | 70.606ms

Comment (flotwig) [reply packages/https-proxy/lib/ca.coffee]: nope, anything less than 2048 results in


```
Error: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small
    at Object.createSecureContext (_tls_common.js:131:17)
    at Server.addContext (_tls_wrap.js:1252:32)
    at /home/person/cypress/packages/https-proxy/lib/server.js:282:23
```

Comment (brian-mann) [reply packages/https-proxy/lib/ca.coffee]: Can you also output / catalog the timings when run from a docker container?

Maybe just add the debug logs in a circle ci build so I have a reference of the differences specifically for CI.

Comment (flotwig) [reply packages/https-proxy/lib/ca.coffee]: alright, i committed the `console.time` logs to ca_spec

**Pull request merged**